rosidl_typesupport_c/include/rosidl_typesupport_c/message_type_support_dispatch.h

```diff
@@ -24,6 +24,16 @@ extern "C"
 {
 #endif
 
+/// Get the message type support handle specific to this identifier.
+/**
+ * This function is a general function used to be called by a generated message's c functionality
```

Comment:
> This function is a general function ...

This part doesn't add any value to the docblock.

> used to be called by a generated message's c functionality

I am not sure what this means? Where is this function being called in the generated message?

Reply:
Removed this sentence in the four files.